styles/themes/theme.tsx

```diff
@@ -23,6 +23,7 @@ const LinkBehavior = forwardRef<HTMLAnchorElement, NextLinkProps>(
 );
 
 const theme = createTheme({
+
```

Comment:
```suggestion
```
Just an empty spot :smile:

Reply:
fixed this

Reply:
Seems that the line you removed was from contact page, but my comment was about that empty line on `theme.tsx`.
Can you check it pls?